api/v1/db/oracledb/player-dao.js

```diff
@@ -18,27 +18,35 @@ const mergeRawPlayers = (rawPlayers) => {
   return mergedRawPlayers;
 };
 
-const getPlayersByGameId = async (id) => {
+const getPlayersByGameId = async (id, query) => {
   const connection = await conn.getConnection();
   try {
     const sqlParams = [id];
+    const gameSqlQuery = `
+    SELECT * FROM GAMES G
+    WHERE G.GAME_ID = :id
+    `;
+    const rawGamesResponse = await connection.execute(gameSqlQuery, sqlParams);
+    const rawGames = rawGamesResponse.rows;
+    if (_.isEmpty(rawGames)) {
+      return undefined;
+    }
     const sqlQuery = `
     SELECT P.PLAYER_ID, P.MEMBER_ID, M.MEMBER_NICKNAME, M.MEMBER_LEVEL, M.MEMBER_EXP_OVER_LEVEL, P.PLAYER_BET, S.STATUS AS PLAYER_STATUS, CN.CARD_NUMBER, CS.SUIT
-    FROM PLAYERS P 
-    INNER JOIN GAMES G ON G.GAME_ID = P.GAME_ID 
+    FROM PLAYERS P
+    INNER JOIN GAMES G ON G.GAME_ID = P.GAME_ID
     INNER JOIN STATUSES S ON S.STATUS_ID = P.STATUS_ID
-    INNER JOIN MEMBERS M ON M.MEMBER_ID = P.MEMBER_ID 
-    LEFT OUTER JOIN PLAYER_CARDS PC ON PC.PLAYER_ID = P.PLAYER_ID 
-    LEFT OUTER JOIN CARDS C ON PC.CARD_ID = C.CARD_ID 
-    LEFT OUTER JOIN CARD_NUMBERS CN ON C.CARD_NUMBER_ID = CN.CARD_NUMBER_ID 
-    LEFT OUTER JOIN CARD_SUITS CS ON C.CARD_SUIT_ID = CS.SUIT_ID  
+    INNER JOIN MEMBERS M ON M.MEMBER_ID = P.MEMBER_ID
+    LEFT OUTER JOIN PLAYER_CARDS PC ON PC.PLAYER_ID = P.PLAYER_ID
+    LEFT OUTER JOIN CARDS C ON PC.CARD_ID = C.CARD_ID
+    LEFT OUTER JOIN CARD_NUMBERS CN ON C.CARD_NUMBER_ID = CN.CARD_NUMBER_ID
+    LEFT OUTER JOIN CARD_SUITS CS ON C.CARD_SUIT_ID = CS.SUIT_ID
     WHERE G.GAME_ID = :id
     `;
```

Comment:
To format a multiline string, can you follow the following format? https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript

Example: https://github.sig.oregonstate.edu/ecs-data/students-api-contrib/blob/master/contrib.js#L2-L8

Reply:
I am not sure what you are expecting for the format.

Reply:
The SQL text should be indented.